### PR TITLE
About page to describe project

### PR DESCRIPTION
**Pull request by githubstudent**
This change brings in the about.txt page

_NOTE:_ It may later make sense to make this an html page for better viewing.

### CODE REVIEW

File: about.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This is some information about the project
+
+It's slightly different than the README, because it's just describing the project in more of a long formed way.
+
+This file was created by @githubstudent
```
Comment (githubstudent) [inline about.txt:5]: This should maybe link to the @githubstudent account

**Comment by githubstudent** [comment]
I think this look good

File: about.txt (hunk 1 above)
Comment (githubteacher) [inline about.txt:5]: This looks good to me for now

File: about.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+This is some information about the project
```
Comment (githubteacher) [inline about.txt:1]: This is not needed.

File: README.md [hunk 3]
```diff
@@ -1 +1,7 @@
 # Class Demo
```
Comment (matthewmccullough) [inline README.md:1]: Comment on line 1 @githubstudent

File: about.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (matthewmccullough) [inline about.txt:2]: And another

**Pull request merged**